LexicalAnalyzer.c

```diff
@@ -0,0 +1,488 @@
+// Lexical Analysis wSQLx
+// Nick Jones
+// 11/18/15
+
+// includes
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+//#include <sys/time.h>
+#include <ctype.h>
+
+// defines
+#define     BUFSIZE    200
+
+// Structs
+typedef struct token {              // token link list node
+    char    type[10];               // can be NUM, ID, KW, SYM
```

Comment:
You can force the compiler to make this only be a NUM, ID, KW or SYM. See https://github.com/JonathanFrias/cop4710-sequel/blob/master/common.h#L33